Formula/hsd.rb

```diff
@@ -0,0 +1,22 @@
+require "language/node"
+
+class Hsd < Formula
+  desc "Handshake Daemon & Full Node"
+  homepage "https://handshake.org"
+  url "https://github.com/handshake-org/hsd/archive/v2.1.2.tar.gz"
+  sha256 "4fc39ee5ca2b91186847a31b3244ae5d57235720ac40812cf752bb687c0833cf"
+
+  depends_on "python" => :build
+  depends_on :xcode => :build
```

Comment:
What does it need xcode for?

Reply:
Doesn't xcode include clang? There is native code that needs to be compiled. Without it, we noticed an error on a user's machine that did not yet ran the command: `$ xcode-select --install`.

Is this correct?

Reply:
`xcode-select --install` installs the command line tools rather than Xcode.app.

`depends_on :xcode` would disallow the command line tools from being used.

Reply:
Thank you for the feedback, I am not an active OSX developer so this is very helpful. Will remove `depends_on :xcode`